### PR TITLE
Adjust {Rest, Realtime}.channels API to spec.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

**Comment by mattheworiordan** [comment]
Overall looks good, one comment in regards to the `channelName` attribute

File: ably-ios/ARTChannels.m [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+#import "ARTRestChannel.h"
+
+@interface ARTChannels() {
+    __weak ARTRest *_rest;
+    __weak id<ARTChannelsDelegate> _delegate;
+}
+
+@end
+
+// FIXME: RestChannelCollection
```
Comment (ricardopereira) [inline ably-ios/ARTChannels.m]: Can we remove this?

**Comment by ricardopereira** [comment]
LGTM

File: ably-ios/ARTChannels.m (hunk 1 above)
Comment (tcard) [inline ably-ios/ARTChannels.m]: Fixed at dd7b7c47c10310cfdb0aa12fa9f4c58d29c30ff8.

**Pull request merged**